### PR TITLE
Added CSS rules for column-container.

### PR DESCRIPTION
**Pull request by felixxm**
Follow up to 21133bb08f5418bd5c8d29cb8045d53bc8cd105c.

### CODE REVIEW

**Comment by felixxm** [comment]
I forgot to commit CSS changes :facepalm:

**Review by bmispelon — APPROVED** [review]
This time I actually tested the PR locally and it looks like it should.

One thing I want to note, but I don't know if it's worth delaying this fix: `trackhacks.css` was originally intended (as far as I remember) as a collection of rules to counteract Trac's builtin CSS.
For the rest of the site, we're supposed to share the same `output.css` between `code.djangoproject.com` and `www.djangoproject.com`.
Ideally we should be able to copy the latest `output.css` version into here, but that might require more whac-a-mole fixes considering the two were last synced several years ago.

TLDR: I think the correct fix is to bring the newest version of `output.css` from `djangoproject.com`, but if that's too big of a job, I think it's acceptable to merge this PR instead

**Comment by felixxm** [comment]
> TLDR: I think the correct fix is to bring the newest version of `output.css` from `djangoproject.com`, but if that's too big of a job, I think it's acceptable to merge this PR instead

To do this, we would need to copy at least the entire `_style.scss` :confused:

**Comment by felixxm** [comment]
> > TLDR: I think the correct fix is to bring the newest version of `output.css` from `djangoproject.com`, but if that's too big of a job, I think it's acceptable to merge this PR instead
> 
> To do this, we would need to copy at least the entire `_style.scss` 😕

Also `_utils.scss` now depends on `_dark-mode.scss` etc. I think we need to clean up a few things in the `djangoproject/scss` before we can use them here.
